tabs/system-setup/5-samba-ssh-setup.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/sh -e
+
+# Load common script functions
+. ../common-script.sh  
+
+# Function to install packages based on the package manager
+install_package() {
+    PACKAGE=$1
+    if ! command_exists "$PACKAGE"; then
+        case "$PACKAGER" in
+            pacman)
+                $ESCALATION_TOOL "$PACKAGER" -S --noconfirm "$PACKAGE"
+                ;;
+            *)
+                $ESCALATION_TOOL "$PACKAGER" install -y "$PACKAGE"
+                ;;
+        esac
+    else
+        echo "$PACKAGE is already installed."
+    fi
+}       
+
+# Function to setup and configure SSH
+setup_ssh() {
+    printf "%b\n" "${YELLOW}Setting up SSH...${RC}"
+
+    # Detect package manager and install appropriate SSH package
+    if [ "$PACKAGER" = "apt-get" ]; then
+        install_package openssh-server
+        SSH_SERVICE="ssh"
+    elif [ "$PACKAGER" = "pacman" ]; then
+        install_package openssh
+        SSH_SERVICE="sshd"
+    else
+        install_package openssh-server
+        SSH_SERVICE="sshd"
+    fi
```

Comment:
1. 
```suggestion
    case "$PACKAGER" in
    "apt-get")
        install_package openssh-server
        SSH_SERVICE="ssh"
        ;;
    "pacman")
        install_package openssh
        SSH_SERVICE="sshd"
        ;;
    *)
        install_package openssh-server
        SSH_SERVICE="sshd"
        ;;
    esac
```
2. This could be merged with https://github.com/ChrisTitusTech/linutil/blob/ea9d6d7ddc04297d69e268ebd40f1a1416c11207/tabs/system-setup/5-samba-ssh-setup.sh#L7-L21